cluster/gce/gci/configure-helper.sh

```diff
@@ -2286,6 +2286,23 @@ EOF
   fi
 }
 
+# A helper function to set up a custom yaml for a k8s addon.
+#
+# $1: addon category under /etc/kubernetes
+# $2: manifest source dir
+# $2: manifest file
```

Comment:
nit: this is $3 and the following line is $4

Reply:
Good catch! Fixed.